pkg/util/selector_test.go

```diff
@@ -353,7 +355,7 @@ func TestClusterMatches(t *testing.T) {
 			want: false,
 		},
 		{
-			name: "[case 1] fields not matched",
+			name: "[case 5] fields not matched",
```

Comment:
Seems "[case 4]" has been omitted.

Reply:
You are very careful. Thank you for pointing it out，I have modified it